index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alfred Portfolio Website</title>
+  </head>
+  <body>
+    <main class="main">
+      <header class="header">
+        <a class="header-logo" href="#">Alfred</a>
+        <a href="javascript:void(0);" class="header-humbugger-icon">
+          <i class="fa fa-bars"></i>
+        </a>
+      </header>
+      <section class="developer">
+        <div class="developer-info">
+          <img
+            class="developer-info-shadow"
+            src="./images/header@2x.png"
+            alt=""
+          />
+          <p class="developer-info-name">Laura Jong</p>
+          <p class="developer-info-title">Software Developer</p>
+        </div>
+        <div class="developer-about">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
+        <div>
+          <hr class="developer-hr-center" />
+        </div>
+
+        <div class="developer-experience">
+          <div class="developer-experience-container">
+            <h5 class="developer-experience-header">Language</h5>
+            <ul class="developer-experience-list">
+              <li class="developer-experience-list-item">Javascript</li>
+              <li class="developer-experience-list-item">Ruby</li>
+              <li class="developer-experience-list-item">HTML</li>
+              <li class="developer-experience-list-item">CSS</li>
+            </ul>
+          </div>
+          <div class="developer-experience-container">
+            <img
+              class="developer-experience-shadow"
+              src="./images/header@2x.png"
+              alt=""
+            />
```

Comment:
- [ ] Kindly add image as a background image with CSS and not as img, please do this as this is part of the project requirement